studybuddies/src/app/components/tutor/create-class-tutor/create-class-tutor.component.ts

```diff
@@ -1,4 +1,8 @@
 import { Component, OnInit } from '@angular/core';
+import {RoomService} from "../../../services/class.service";
+import { NgForm } from "@angular/forms";
+import { Class } from "../../../models/class";
+import { Router } from '@angular/router';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203289)
```suggestion
import { Router } from "@angular/router";
```